content/docs/iac/concepts/resources/dynamic-providers.md

```diff
@@ -751,13 +762,14 @@ export class Label extends pulumi.dynamic.Resource {
 {{% choosable language python %}}
 
 ```python
+import pulumi
 from pulumi import ComponentResource, export, Input, Output
-from pulumi.dynamic import Resource, ResourceProvider, CreateResult, UpdateResult
+from pulumi.dynamic import Resource, ResourceProvider, CreateResult, UpdateResult, ConfigureRequest
 from typing import Optional
 from github import Github, GithubObject
 
-auth = "<auth token>"
-g = Github(auth)
+config = pulumi.Config()
+auth = config.require_secret("githubToken")
```

Comment:
I think we can remove these 2 lines, and then use `g = Github(self.auth.get())` below

```suggestion
```

Reply:
I removed these two lines.